bundletool/retryhttp.go

```diff
@@ -0,0 +1,23 @@
+package bundletool
+
+import (
+	"github.com/bitrise-io/go-utils/log"
+	retryablehttp "github.com/hashicorp/go-retryablehttp"
+)
+
+// RetryLogAdaptor adapts the retryablehttp.Logger interface to the go-utils logger.
+type RetryLogAdaptor struct{}
+
+// Printf implements the retryablehttp.Logger interface
+func (*RetryLogAdaptor) Printf(fmtStr string, vars ...interface{}) {
+	log.Debugf(fmtStr, vars...)
+}
+
+// NewRetryableClient returns a retryable HTTP client
+func NewRetryableClient() *retryablehttp.Client {
```

Comment:
This should go to the go-utils too

Reply:
What  is the default "backoff" strategy?

Reply:
It is an exponential backoff (limited by min/max settings): https://github.com/hashicorp/go-retryablehttp/blob/02c1586c8f14be23e7eeb522f1094afbabf45e93/client.go#L472